### PR TITLE
Adapt stack-to-nix flags to new synthax.

### PR DESCRIPTION
**Pull request by jbgi**

### CODE REVIEW

File: docs/nix-toolification.org [hunk 1]
```diff
@@ -243,7 +243,7 @@ nix build -f default.nix _iohk.nix-tools.regeneratePackages --out-link regen-pkg
 This will install ~nix-tools~ and call
 
 #+BEGIN_SRC
-stack-to-nix -o nix stack.yaml > nix/.stack-pkgs.nix~.
+stack-to-nix --output nix --stack-yaml stack.yaml > nix/.stack-pkgs.nix~.
```
Comment (angerman) [inline docs/nix-toolification.org]:
```suggestion
stack-to-nix --output nix --stack-yaml stack.yaml
```

File: nix-tools-regenerate.nix [hunk 2]
```diff
@@ -32,7 +32,7 @@ in
     }
     trap cleanup EXIT
 
-    stack-to-nix -o nix stack.yaml > "$dest.new"
+    stack-to-nix --output nix > "$dest.new"
```
Comment (angerman) [inline nix-tools-regenerate.nix]:
```suggestion
    stack-to-nix --output nix
```

Comment (angerman) [reply nix-tools-regenerate.nix]: I don't think `stack-to-nix` generates anything on STDOUT anymore.

But then the two `mv` and `echo` lines below make no sens anymore either.

**Comment by angerman** [comment]
@jbgi I believe the whole `dest` and `dest.new` logic in ` nix-tools-regenerate.nix` doesn't make much sense now anymore? Do you agree?

**Comment by jbgi** [comment]
> @jbgi I believe the whole `dest` and `dest.new` logic in ` nix-tools-regenerate.nix` doesn't make much sense now anymore? Do you agree?

Yes. I am working on a fix right now.

**Pull request merged**